project.md

```diff
@@ -58,11 +78,19 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+My goal is to spend 20% of my time on systems and the other 80% of the time on language design. I think that this split allows me to pursue both areas while emphasizing the latter (as this is a programming languages class, after all).
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+I think that this idea is reasonably sized, as it will provide some interesting implementation challenges while keeping the project focused on language design. I also think that this domain is well-suited for scaling the scope of the project up or down depending on unexpected successes or roadblocks. For example, if things are going swimmingly, I can tackle some of the extended goals outlined in the *Grading* section of my contract. On the other hand, if things go awry on the implementation I will switch to interfacing with an existing text-based game project (as also discussed in my contract).
```

Comment:
If you do have to do the latter, it's still a good thing that you're keeping your project using especially simple syntax for writers, which will help set it apart from the existing DSL you're interfacing with.